### DISCUSSION TITLE
UX/UI

### DISCUSSION THREAD

**Discussion by ghost** [lllyasviel/Fooocus#1965] [General]
Fantastic work on developing this app! I'm thoroughly impressed by what you've achieved. Should you require any assistance with UX/UI design, I would be delighted to offer my expertise and collaborate

**Comment by mashb1t** [comment]
Happy to hear you're having a good time with Fooocus!
Feel free to check the existing PRs and provide your input if you see anything of interest where you have suggestions for.

**Comment by DylanOpet** [comment]
I'd also love to help out and collaborate as I explore the tool more. 

Cheers,
[Dylan](https://dylanopet.me/)

**Comment by ghost** [comment]
I did notice the presence of the LCM preset, but I'm not sure if it's fully implemented yet or just a future placeholder. If you plan to implement it later on, where exactly (and how) are you planning to do it? Will it be an extra option on the default "settings" screen?

**Reply by mashb1t** [reply]
LCM has fully been implemented and is usable without any issues (Extreme Speed performance setting)

**Comment by ghost** [comment]
I've made some improvements following the 'less is more' principle, as I believe the style section was a bit overwhelming as it was. The screens should be self-explanatory for developers, but if you have any questions, please feel free to ask. I hope you find these changes useful. 

![Frame 1](https://github.com/lllyasviel/Fooocus/assets/125589442/0d231cd7-1d0d-4971-9dab-20e8964f96f8)

![Frame 2](https://github.com/lllyasviel/Fooocus/assets/125589442/549a5820-5ed4-410e-b316-50c9f8e27538)

**Reply by mashb1t** [reply]
the column layout has already been implemented in https://github.com/lllyasviel/Fooocus/pull/1883, see screenshots in https://github.com/lllyasviel/Fooocus/issues/1770#issuecomment-1887719446.
Sorting is done using a custom solution for most recent first, but your proposal looks good. Gradio just does not allow split lists of selected/unselected options as well as grouping in RadioGroup elements.

**Comment by ghost** [comment]
I also think you can replace the "radio" buttons in the advanced img2img section with a single dropdown. This will make the screen look less "busy" and open the possibility to add more options in the future without worrying about screen real estate availability.

![Untitled-1](https://github.com/lllyasviel/Fooocus/assets/125589442/4527cc6b-9265-43e7-b199-7a765af167f7)
